noggin/tests/unit/conftest.py

```diff
@@ -113,3 +113,17 @@ def logged_in_dummy_user(client, dummy_user):
     ipa.logout()
     with client.session_transaction() as sess:
         sess.clear()
+
+
+@pytest.fixture
+def dummy_user_with_2fa(client, dummy_user):
```

Comment:
I've merged a change to master that has a fixture pretty similar to this one (called `dummy_user_with_otp`), could you please rebase and use it?